plugins/search/src/components/DefaultResultListItem/DefaultResultListItem.stories.tsx

```diff
@@ -14,13 +14,13 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import { Button } from '@backstage/core-components';
 import { Grid } from '@material-ui/core';
 import FindInPageIcon from '@material-ui/icons/FindInPage';
 import GroupIcon from '@material-ui/icons/Group';
-import { Button } from '@backstage/core-components';
-import { DefaultResultListItem } from '../index';
```

Comment:
sorry for the rearrange here out of old habit - i just was out for fixing `index` and "dot" imports actually